maps/random/corinthian_isthmus_ii.js

```diff
@@ -321,20 +372,24 @@ if (!isNomad())
 				0.1,
 				-0.6),
 			[
-				new LayeredPainter([tRoad, tDirt, tRoad], [1, 3]),
-				new SmoothElevationPainter(ELEVATION_MODIFY, heightPath, 4),
+				new LayeredPainter([tRoad, tDirt, tRoad], [2, 4]),
+				new SmoothElevationPainter(ELEVATION_MODIFY, heightOffsetRoad, 4),
 				new TileClassPainter(clRoad)
 			],
 			[
-				// new NearTileClassConstraint(g_TileClasses.water, 0),
-				// avoidClasses(g_TileClasses.water, 0)
-				// stayClasses(g_TileClasses.land, 2)
+				roadConstraint
```

Comment:
@Jammyjamjamman Doing it this way, 3 or 4 roads get drawn, whereas before, when using 'avoidClasses' there, no roads get drawn.

You can several lines above how `roadConstraint` is defined...